charts/values.example.ce.yaml

```
@@ -34,7 +34,7 @@ dex:
 
   # The "kubermaticIssuer" client is used for providing OIDC access to User Clusters.
   # This configuration is optional, used if the "enableOIDCKubeconfig: true" option is used in KubermaticSetting.
-  # More about this configuration at https://docs.kubermatic.com/kubermatic/master/tutorials-howtos/oidc-provider-configuration/share-clusters-via-delegated-oidc-authentication/
+  # More about this configuration at https://docs.kubermatic.com/kubermatic/mainorials-howtos/oidc-provider-configuration/share-clusters-via-delegated-oidc-authentication/
```

Something went wrong here.

```suggestion
  # More about this configuration at https://docs.kubermatic.com/kubermatic/main/tutorials-howtos/oidc-provider-configuration/share-clusters-via-delegated-oidc-authentication/
```